book/02-git-basics/sections/remotes.asc

```diff
@@ -179,7 +179,7 @@ $ git remote show origin
 ----
 
 It lists the URL for the remote repository as well as the tracking branch information.
-The command helpfully tells you that if you're on the `master` branch and you run `git pull`, it will automatically merge in the `master` branch on the remote after it fetches all the remote references.
+The command helpfully tells you that if you're on the `master` branch and you run `git pull`, it will automatically merge in the `master` branch on the local after it fetches all the remote references.
```

Comment:
Not sure this is more correct. The idea we're trying to convey is that _on your local repo_, you'll end up with a branch that has all of your changes and will merge in any changes that have happened on the remote. What you've got here makes it sound like your local `master` will be merged with… the local `master`.

I'm in agreement with you here that something needs to change to be more clear. We're in chapter 2, so we still need to hand-wave some technical details to make sure the beginner reader gets the main idea. What about this instead:

```suggestion
The command helpfully tells you that if you're on the `master` branch and you run `git pull`, it will automatically merge the remote's `master` branch into the local one after it has been fetched.
```

Reply:
@ben ok, ack the idea